pymc/tests/test_variational_inference.py

```diff
@@ -152,15 +137,14 @@ def test_init_groups(three_var_model, raises, grouping):
             {
                 MeanFieldGroup: (["one"], {}),
                 FullRankGroup: (["two"], {}),
-                NormalizingFlowGroup: (["three"], {"flow": "scale-hh*2-planar-radial-loc"}),
+                MeanFieldGroup: (["three"], {}),
```

Comment:
Hi @ferrine running `test_logq_globals` after replacing `NormalizingFlowGroup` with `MeanFieldGroup` gives the bellow error
```
pymc.variational.opvi.GroupError: No approximation is specified for the rest variables
```
can I removing this particular test case or there any way to fix this

Reply:
Same for `test_pickle_approx`